src/toil/utils/toilKill.py

```diff
@@ -41,11 +41,12 @@ def main() -> None:
         logger.info("All jobs SHOULD have been killed")
     # otherwise, kill the pid recorded in the jobstore
     else:
-        pid_log = os.path.join(os.path.abspath(config.jobStore), 'pid.log')
+        jobStore = Toil.resumeJobStore(config.jobStore)
+        pid_log = os.path.join(jobStore.sharedFilesDir, 'pid.log')
         with open(pid_log) as f:
             pid2kill = f.read().strip()
         try:
-            os.kill(int(pid2kill), signal.SIGKILL)
+            os.kill(int(pid2kill), signal.SIGTERM)
```

Comment:
It would be nice if we could pick the signal from the command line, or send a kill if it's still running after a while, or if we could hold off on declaring the process "successfully terminated" until it actually dies. But those might all be separate feature requests; it's probably good enough to just make `toil kill` work at all again.

Reply:
The simplest thing to start would be a CLI option to set the signal type, with SIGTERM being the default. Kind of like normal `kill` has.

Being super smart about checking if it stopped in a reasonable amount of time would be a harder design problem.

But neither needs to be part of *this* PR.